Doc/library/asyncio-task.rst

```diff
@@ -210,6 +210,10 @@ is :meth:`loop.run_in_executor`.
 Running an asyncio Program
 ==========================
 
+**Source code:** :source:`Lib/asyncio/runners.py`
+
+-------------------------------------------------
```

Comment:
Why do we need a line here?  I understand why it's added when a link is top level, but here it seems a bit excessive.

Reply:
@1st1 It's not needed by any means, I just thought it looked better with the line once I saw it rendered in Sphinx (I originally omitted the line for the non-header ones, as you suggested doing):

With line:
![image](https://user-images.githubusercontent.com/44193521/66439676-6eb79200-e9fe-11e9-9677-6740beb61209.png)

Without line:
![image](https://user-images.githubusercontent.com/44193521/66439761-b2aa9700-e9fe-11e9-8799-b22eb3adc2de.png)

Does seeing the difference in how it renders change anything or would you still prefer to remove the line? Either way is fine for me.

Reply:
You can add a `.. note::` at the end of the docs about asyncio.run:

```
.. note::
   The source code of the function can be found here: :source:`Lib/asyncio/runners.py`
```

Reply:
Done. It now renders as:

![image](https://user-images.githubusercontent.com/44193521/66608872-8c633380-eb85-11e9-817e-30a1168c845e.png)